Gemfile.lock

```diff
@@ -4,22 +4,22 @@ PATH
     high_voltage (2.4.0)
 
 GEM
-  remote: http://rubygems.org/
+  remote: https://rubygems.org/
   specs:
-    actionpack (4.2.0)
-      actionview (= 4.2.0)
-      activesupport (= 4.2.0)
-      rack (~> 1.6.0)
+    actionpack (4.2.5.2)
```

Comment:
I'd rather have tests and development environment install all the latest allowed versions for the dependencies, to make sure they are properly declared. Is there a reason why we'd prefer to keep it?